Localizations/en_US.json

```diff
@@ -173,39 +173,39 @@
+    "shader_asset_format_dummy_description": "Export the shader as a dummy shader. Although it preserves data like the Properties and Fallback, it uses general, opaque shader code.",
+    "shader_asset_format_yaml": "YAML Asset",
+    "shader_asset_format_yaml_description": "Export the shader as a YAML asset. Experimental and only usable for viewing in the editor. The editor can randomly corrupt these files.",
     "skip_streaming_assets": "Skip StreamingAssets Folder",
     "sprite_export_title": "Sprite Export Format",
     "sprite_format_native": "Unity",
-    "sprite_format_native_description": "Export in the unity sprite format. Cannot be viewed outside of unity.",
+    "sprite_format_native_description": "Export in the Unity sprite format. Cannot be viewed outside of Unity.",
     "sprite_format_texture": "Texture",
-    "sprite_format_texture_description": "Export as an image of the Sprite Sheet. Can be viewed outside of unity, but slower to export.",
-    "sprite_format_yaml": "Yaml",
-    "sprite_format_yaml_description": "Export as yaml assets which can be viewed in the editor. This is the only mode that ensures a precise recovery of all metadata of sprites.",
+    "sprite_format_texture_description": "Export as an image of the sprite sheet. Can be viewed outside of Unity, but slower to export.",
+    "sprite_format_yaml": "YAML",
+    "sprite_format_yaml_description": "Export as YAML assets which can be viewed in the editor. This is the only mode that ensures a precise recovery of all metadata of sprites.",
     "submesh_count": "Submesh Count",
     "success": "Success!",
     "terrain_export_title": "Terrain Export Format",
     "terrain_format_heatmap": "Heightmap",
-    "terrain_format_heatmap_description": "Export a heatmap of the height of the terrain at each location. Only really useful if you don't care about the details or having the terrain in 3D.",
+    "terrain_format_heatmap_description": "Export a heat map of the height of the terrain at each location. Only really useful if you don't care about the details or having the terrain in 3D.",
     "terrain_format_mesh": "3D Mesh",
     "terrain_format_mesh_description": "Export the terrain as a 3D mesh in GLB format, suitable for viewing with a wide range of 3D editors.",
-    "terrain_format_native": "Yaml",
-    "terrain_format_native_description": "Export in the native unity terrain format. Most useful option if you plan to re-import back into unity.",
+    "terrain_format_native": "YAML",
+    "terrain_format_native_description": "Export in the native Unity terrain format. Most useful option if you plan to re-import back into Unity.",
     "text_asset_export_title": "TextAsset Export Format",
     "text_asset_format_binary": "Bytes",
-    "text_asset_format_binary_description": "Export the raw bytes of the text asset with a .bytes extension.",
+    "text_asset_format_binary_description": "Export the raw bytes of the text asset as a BYTES file.",
```

Comment:
I initially wanted to reject this change, but then I looked at the others and noticed that this is more consistent with them. The only change I would make to your wording is "the raw data" instead of "the raw bytes."

Reply:
Sure, I'll change that!